InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/RamseteSpecial.java

```diff
@@ -0,0 +1,175 @@
+/*----------------------------------------------------------------------------*/

```

Comment:
Is this file being used, or was it just for testing?

Reply:
No, this was just a test for running the controller past the time-limit of the trajectory.